.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,10 +25,17 @@
 1. 
 2. 
 
+#### Testing this PR requires running the following commands:
+Action |  Command  | Required
+| --- | --- | ---
+clear cache | php app/console cache:clear | 
+regenerate assets | php app/console mautic:assets:generate | 
+update db schema | run command php app/console doctrine:schema:update --force | 
```

Comment:
Actually, this schema update should never be used (theoretically). Since each PR modifying the schema should have the migration file, can you switch it to `app/console doctrine:migrations:migrate`? Also the migrations can contain some data manipulation which would populate the new schema.